tools/restatectl/src/commands/log/reconfigure.rs

```diff
@@ -12,6 +12,7 @@ use std::num::{NonZeroU32, NonZeroU8};
 
 use anyhow::Context;
 use cling::prelude::*;
+use restate_cli_util::{c_eprintln, c_println};
```

Comment:
This could be grouped with the other restate types below.

PS. I think we should switch to `rustfmt` nightly, and have these be auto-reordered everywhere so we can stop thinking about it.

Reply:
Ah yes, thank you I will update that.